trio/_core/tests/test_multierror.py

```diff
@@ -150,21 +141,10 @@ def handle_ValueError(exc):
         else:
             return exc
 
-    filtered_excs = MultiError.filter(handle_ValueError, excs)
+    filtered_excs = pytest.deprecated_call(MultiError.filter, handle_ValueError, excs)
     assert isinstance(filtered_excs, NotHashableException)
 
 
-def test_traceback_recursion():
```

Comment:
I think this is still a good test for wherever you're doing your traceback monkeypatching; I wouldn't want to remove it here unless it's been added in `exceptiongroup` and in CPython. Same comment applies to the big chunk of removed code further below in this file.

Reply:
I'll look into adopting this in `exceptiongroup`.

Reply:
I'm all in favor of moving tests of `exceptiongroup` functionality to `exceptiongroup`, but it's important to me that we not throw tests away without moving them.